src/iocore/net/SSLDiags.cc

```diff
@@ -69,7 +71,29 @@ increment_ssl_client_error(unsigned long err)
   case SSL_R_TLSV1_ALERT_UNKNOWN_CA:
     Metrics::Counter::increment(ssl_rsb.user_agent_unknown_ca);
     break;
+  case SSL_R_DECRYPTION_FAILED_OR_BAD_RECORD_MAC:
+    Metrics::Counter::increment(ssl_rsb.user_agent_decryption_failed_or_bad_record_mac);
+    break;
+  case SSL_R_HTTP_REQUEST:
+    Metrics::Counter::increment(ssl_rsb.user_agent_http_request);
+    break;
+  case SSL_R_INAPPROPRIATE_FALLBACK:
+    Metrics::Counter::increment(ssl_rsb.user_agent_inappropriate_fallback);
+    break;
+  case SSL_R_NO_SHARED_CIPHER:
+    Metrics::Counter::increment(ssl_rsb.user_agent_no_shared_cipher);
+    break;
+#ifndef OPENSSL_IS_BORINGSS
```

Comment:
Typo - missing L at the end.

I'd check availability of `SSL_R_VERSION_TOO_HIGH`. That'd work for AWS-LC as well.

Reply:
Yeah, that's a better idea.